src/lib.rs

```diff
@@ -74,10 +79,18 @@ pub fn reverse_proxy_filter(
         .boxed()
 }
 
+pub fn query_params(
```

Comment:
Nitpicking here, could we rename this to `query_params_filter`?

Reply:
Done, added some docs too